torch/csrc/jit/codegen/cuda/kernel.h

```diff
@@ -19,6 +19,9 @@ namespace fuser {
 // TODO(kir): const node ptrs
 //
 struct KernelSummary {
+  //! List of WAR syncs
```

Comment:
nit: I prefer to expand the acronyms in comments (ex. `List of WAR (Write-After-Read) synchronization barriers`)

Reply:
:+1: